theme/static/js/hs-vue/left-header-app.js

```diff
@@ -4,23 +4,61 @@ let leftHeaderApp = new Vue({
     data: {
         owners: [
             {
-                user_type: "initial",
-                access: "initial",
-                id: "initial",
-                pictureUrl: "initial",
-                best_name: "initial",
-                user_name: "initial",
-                can_undo: "initial",
-                email: "initial",
-                organization: "initial",
-                title: "initial",
-                contributions: "initial",
+                user_type: "owner_1",
+                access: "owner_1",
+                id: "owner_1",
+                pictureUrl: "owner_1",
+                best_name: "owner_1",
+                user_name: "owner_1",
+                can_undo: "owner_1",
+                email: "owner_1",
+                organization: "owner_1",
+                title: "owner_1",
+                contributions: "owner_1",
                 subject_areas: ["one", "two", "three"],
                 identifiers: [],
-                state: "initial",
-                joined: "initial",
+                state: "owner_1",
+                joined: "owner_1",
+            },
+            {
+                user_type: "owner_2",
```

Comment:
Why add another owner here?  I'm not following.

Reply:
This was placeholder data that I forgot to clean up. I'll take care of it.

Reply:
@sblack-usu Looks like this is already cleaned up. Are you looking at an outdated version of these files?